### PR TITLE
ApiDoc: Add response description to WallabagRestController

### PR DESCRIPTION
**Pull request by caspermeijn**
| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Documentation | yes
| Translation   | no
| CHANGELOG.md  | no
| License       | MIT

Add response description to the API doc of WallabagRestController.
I started with this controller as it is the simplest.
I move the creation of the response to a separate class to document the response.
I don't have much PHP experience, so I am not sure whether this is good practice.

### CODE REVIEW

File: src/Wallabag/ApiBundle/Controller/WallabagRestController.php [hunk 1]
```diff
@@ -14,17 +15,61 @@
 use Symfony\Component\Security\Core\Exception\AccessDeniedException;
 use Wallabag\UserBundle\Entity\User;
 
+class ApplicationInfo
```
Comment (nicosomb) [inline src/Wallabag/ApiBundle/Controller/WallabagRestController.php:18]: could you please create a new file for this class?

**Review by nicosomb — CHANGES_REQUESTED** [review]

File: src/Wallabag/ApiBundle/Controller/WallabagRestController.php (hunk 1 above)
Comment (caspermeijn) [reply src/Wallabag/ApiBundle/Controller/WallabagRestController.php]: Fixed

**Comment by caspermeijn** [comment]
I rebased to master, but it seems like these instructions don't work anymore https://github.com/wallabag/wallabag/blob/master/.github/CONTRIBUTING.md

```
phpstan/extension-installer: Extensions installed
> Incenteev\ParameterHandler\ScriptHandler::buildParameters
Updating the "app/config/parameters.yml" file
Script Incenteev\ParameterHandler\ScriptHandler::buildParameters handling the post-cmd event terminated with an exception

                                                                               
  [Symfony\Component\Yaml\Exception\ParseException]                            
  The reserved indicator "%" cannot start a plain scalar; you need to quote t  
  he scalar at line 9 (near "database_path: %kernel.project_dir%/data/db/wall  
  abag.sqlite").                                                               
                                                                               

install [--prefer-source] [--prefer-dist] [--prefer-install PREFER-INSTALL] [--dry-run] [--dev] [--no-suggest] [--no-dev] [--no-autoloader] [--no-progress] [--no-install] [-v|vv|vvv|--verbose] [-o|--optimize-autoloader] [-a|--classmap-authoritative] [--apcu-autoloader] [--apcu-autoloader-prefix APCU-AUTOLOADER-PREFIX] [--ignore-platform-req IGNORE-PLATFORM-REQ] [--ignore-platform-reqs] [--] [<packages>]...

ERROR: 1
```

**Comment by j0k3r** [comment]
Could you tell what it really adds to the documentation?
Maybe show us a screenshot of the doc before/after

**Comment by caspermeijn** [comment]
I rebased the branch to resolve the conflict.

> Could you tell what it really adds to the documentation? Maybe show us a screenshot of the doc before/after

I am not able to run the server locally due to #6173 , so I have not been able to provide screenshots.

This adds documentation of the response to `/api/info` and `/api/version` calls. This includes an example of the json.

**Comment by caspermeijn** [comment]
# /api/version
## Before
Schermafdruk van 2022-12-23 10-49-44

## After

Schermafdruk van 2022-12-23 10-51-15

Schermafdruk van 2022-12-23 10-51-29

# /api/info
## Before
Schermafdruk van 2022-12-23 10-50-07

## After 
Schermafdruk van 2022-12-23 10-51-46

Schermafdruk van 2022-12-23 10-51-55

**Comment by caspermeijn** [comment]
@nicosomb @j0k3r I rebased this branch to the current master. Please let me know if you have any problems with the current state of the PR.

**Comment by caspermeijn** [comment]
The failing CI test seems to be on the master branch. It seems that all recent PRs have the same failing test.

**Review by nicosomb — APPROVED** [review]

**Comment by caspermeijn** [comment]
I rebased the changes and verified that the docs worked as expected.

**Review by nicosomb — APPROVED** [review]

**Review by j0k3r — APPROVED** [review]

**Pull request merged**